EmotiBit.cpp

```diff
@@ -1101,10 +1102,10 @@ bool EmotiBit::setupSdCard()
 		Serial.print(i);
 		Serial.print(",");
 		// begin function initializes SPI at MAX speed by deafult. check /src/SpiDriver/SdSpiDriver.h for details on the constructor
-#ifdef ADAFRUIT_FEATHER_M0
-		if (SD.begin(EmotiBitVersionController::SD_CARD_CHIP_SEL_PIN))
-#elif defined ARDUINO_FEATHER_ESP32
+#if defined ARDUINO_FEATHER_ESP32
```

Comment:
conform to `#ifdef ARDUINO_FEATHER_ESP32`
That seems to be the pattern we are following.